packages/sdk/src/swap/types.ts

```diff
@@ -66,7 +66,7 @@ export interface DepositAddressRequest extends QuoteRequest {
   ccmParams?: CcmParams;
   maxBoostFeeBps?: number;
   srcAddress?: string;
-  fillOrKillParams?: FillOrKillParamsWithMinPrice;
+  fillOrKillParams: FillOrKillParamsWithMinPrice;
```

Comment:
mandatory in the protocol since https://github.com/chainflip-io/chainflip-backend/pull/5694